### ISSUE TITLE
How to creat a cycle animation（rotate）?

### ISSUE THREAD

**Issue by cssoul** [brentvatne/react-native-animated-demo-tinder#6]
Hello，):
I want to know something about the Animation of React Native.
Referring to your example, I made a demo to show a cycle of animation（rotate）, but failed... o(╯□╰)o
I just find AnimationExperimental can do it. But of course, AnimationExperimental doesn't have infinite property.
do you have others way?

```
AnimationExperimental.startAnimation({
      node: this.refs['rotation'],      
      duration: 200*400,
      easing: 'linear',
      property: 'rotation',
      toValue: 400
});
```

**Comment by brentvatne** [comment]
Check this out: https://gist.github.com/brentvatne/4da16432230c1194a446

**Comment by cssoul** [comment]
thanks :）
At last ,I achieved this effect:
https://gist.github.com/cssoul/eda63b173311a323653b

**Comment by cssoul** [comment]
thanks :）
At last ,I achieved this effect:
https://gist.github.com/cssoul/eda63b173311a323653b

2015-07-22 5:47 GMT+08:00 Brent Vatne notifications@github.com:

> Closed #6
> https://github.com/brentvatne/react-native-animated-demo-tinder/issues/6
> .
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/brentvatne/react-native-animated-demo-tinder/issues/6#event-361660058
> .

## 

 /****************************************************

From  ：张军
旺旺ID：黑幕困兽
E-mail：hbjszj@gmail.com

*****************************************************/
